docs/source/user_guide/tutorials.rst

```diff
@@ -46,7 +46,7 @@ This can be done either by running,
 
 .. code-block:: bash
 
-   $ curl -L https://raw.githubusercontent.com/landlab/landlab/mcflugen/fetch-notebooks-script/notebooks.py | python -
+   $ curl -L https://raw.githubusercontent.com/landlab/landlab/master/notebooks.py | python -
```

Comment:
Don't forget to update the link on line 57 below.

https://github.com/landlab/landlab/blob/master/notebooks.py

Reply:
Whoops! Thanks for catching that.